src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.unit.spec.jsx

```diff
@@ -38,13 +38,14 @@ describe('ApplyForBenefits component', () => {
     };
     view = renderInReduxProvider(<ApplyForBenefits />, {
       initialState,
-      reducer,
+      reducers,
```

Comment:
Oops, this was broken but tests were previously still working. My changes caused the unit tests to blow up because `renderInReduxProvider` was expecting the option object to have a `reducerS` key.